backend/python/app/services/implementations/comment_service.py

```diff
@@ -0,0 +1,34 @@
+from flask import current_app
+
+from ...models import db
+from ...models.comment import Comment
+from ..interfaces.comment_service import ICommentService
+
+
+class CommentService(ICommentService):
+    def __init__(self, logger=current_app.logger):
+        self.logger = logger
+
+    def create_comment(self, comment):
+        try:
+            new_comment = Comment(**comment)
+            comment_index = 0
+
+            comments = Comment.query.filter_by(
+                story_translation_content_id=comment["story_translation_content_id"]
+            )
+
+            if comments.first() is not None:
+                comment_index = (
+                    comments.order_by(Comment.comment_index)[-1].comment_index + 1
```

Comment:
nice! Is there something like `.max()` in sqlalchemy or mysql? I wonder if there's something faster than sorting.